### ISSUE TITLE
Plugin is missing build

### ISSUE THREAD

**Issue by DimitrisRK** [nativescript-community/ui-canvas#20]
### Which platform(s) does your issue occur on?
- Android

### Please, provide the following version numbers that your issue occurs with:

- CLI: 7.0.6
- Cross-platform modules: 7.0.0
- Runtime(s): 7.0.0
- Plugin(s): 4.0.5

### Please, tell us how to recreate the issue in as much detail as possible.

It looks like plugin has android.support issues, just like this case: https://github.com/nativescript-community/gesturehandler/issues/6

**Comment by farfromrefug** [comment]
@DimitrisRK do you get build errors?

**Comment by DimitrisRK** [comment]
Yes, here is the error:
```
Webpack build done!                                                                                                                
F:\my-workspace\ohoapp\platforms\tempPlugin\ui_canvas\src\main\java\com\akylas\canvas\CanvasPath.java:5: error: package android.sup
tion does not exist                                                                                                                
import android.support.annotation.NonNull;                                                                                         
                                 ^                                                                                                 
F:\my-workspace\ohoapp\platforms\tempPlugin\ui_canvas\src\main\java\com\akylas\canvas\CanvasPath.java:6: error: package android.sup
tion does not exist                                                                                                                
import android.support.annotation.Nullable;                                                                                        
                                 ^                                                                                                 
F:\my-workspace\ohoapp\platforms\tempPlugin\ui_canvas\src\main\java\com\akylas\canvas\CanvasPath.java:7: error: package android.sup
tion does not exist                                                                                                                
import android.support.annotation.Size;                                                                                            
                                 ^                                                                                                 
F:\my-workspace\ohoapp\platforms\tempPlugin\ui_canvas\src\main\java\com\akylas\canvas\CanvasPath.java:20: error: cannot find symbol
   public void addLines(@Size(multiple = 2, min = 4) @NonNull float[] points) {                                                    
                         ^                                                                                                         
  symbol:   class Size                                                                                                             
  location: class CanvasPath                                                                                                       
F:\my-workspace\ohoapp\platforms\tempPlugin\ui_canvas\src\main\java\com\akylas\canvas\CanvasPath.java:20: error: cannot find symbol
   public void addLines(@Size(multiple = 2, min = 4) @NonNull float[] points) {                                                    
                                                      ^                                                                            
  symbol:   class NonNull                                                                                                          
  location: class CanvasPath                                                                                                       
F:\my-workspace\ohoapp\platforms\tempPlugin\ui_canvas\src\main\java\com\akylas\canvas\CanvasPath.java:23: error: cannot find symbol
   public void addLines(@Size(multiple = 2, min = 4) @NonNull float[] points, int offset) {                                        
                         ^                                                                                                         
  symbol:   class Size                                                                                                             
  location: class CanvasPath                                                                                                       
F:\my-workspace\ohoapp\platforms\tempPlugin\ui_canvas\src\main\java\com\akylas\canvas\CanvasPath.java:23: error: cannot find symbol
   public void addLines(@Size(multiple = 2, min = 4) @NonNull float[] points, int offset) {                                        
                                                      ^                                                                            
  symbol:   class NonNull                                                                                                          
  location: class CanvasPath                                                                                                       
F:\my-workspace\ohoapp\platforms\tempPlugin\ui_canvas\src\main\java\com\akylas\canvas\CanvasPath.java:26: error: cannot find symbol
   public void addLines(@Size(multiple = 2, min = 4) @NonNull float[] points, int offset, int length) {                            
                         ^                                                                                                         
  symbol:   class Size                                                                                                             
  location: class CanvasPath                                                                                                       
F:\my-workspace\ohoapp\platforms\tempPlugin\ui_canvas\src\main\java\com\akylas\canvas\CanvasPath.java:26: error: cannot find symbol
   public void addLines(@Size(multiple = 2, min = 4) @NonNull float[] points, int offset, int length) {                            
                                                      ^                                                                            
  symbol:   class NonNull                                                                                                          
  location: class CanvasPath                                                                                                       
F:\my-workspace\ohoapp\platforms\tempPlugin\ui_canvas\src\main\java\com\akylas\canvas\CanvasPath.java:32: error: cannot find symbol
   public void addLines(@Size(multiple = 2, min = 4) @NonNull float[] points, int offset, int length, boolean close) {             
                         ^                                                                                                         
  symbol:   class Size                                                                                                             
  location: class CanvasPath                                                                                                       
F:\my-workspace\ohoapp\platforms\tempPlugin\ui_canvas\src\main\java\com\akylas\canvas\CanvasPath.java:32: error: cannot find symbol
   public void addLines(@Size(multiple = 2, min = 4) @NonNull float[] points, int offset, int length, boolean close) {             
                                                      ^                                                                            
  symbol:   class NonNull                                                                                                          
  location: class CanvasPath                                                                                                       
F:\my-workspace\ohoapp\platforms\tempPlugin\ui_canvas\src\main\java\com\akylas\canvas\CanvasPath.java:43: error: cannot find symbol
   public void setLines(@Size(multiple = 2, min = 4) @NonNull float[] points) {                                                    
                         ^                                                                                                         
  symbol:   class Size                                                                                                             
  location: class CanvasPath                                                                                                       
F:\my-workspace\ohoapp\platforms\tempPlugin\ui_canvas\src\main\java\com\akylas\canvas\CanvasPath.java:43: error: cannot find symbol
   public void setLines(@Size(multiple = 2, min = 4) @NonNull float[] points) {                                                    
                                                      ^                                                                            
  symbol:   class NonNull                                                                                                          
  location: class CanvasPath                                                                                                       
F:\my-workspace\ohoapp\platforms\tempPlugin\ui_canvas\src\main\java\com\akylas\canvas\CanvasPath.java:46: error: cannot find symbol
   public void setLines(@Size(multiple = 2, min = 4) @NonNull float[] points, int offset) {                                        
                         ^                                                                                                         
  symbol:   class Size                                                                                                             
  location: class CanvasPath                                                                                                       
F:\my-workspace\ohoapp\platforms\tempPlugin\ui_canvas\src\main\java\com\akylas\canvas\CanvasPath.java:46: error: cannot find symbol
   public void setLines(@Size(multiple = 2, min = 4) @NonNull float[] points, int offset) {                                        
                                                      ^                                                                            
  symbol:   class NonNull                                                                                                          
  location: class CanvasPath                                                                                                       
F:\my-workspace\ohoapp\platforms\tempPlugin\ui_canvas\src\main\java\com\akylas\canvas\CanvasPath.java:49: error: cannot find symbol
   public void setLines(@Size(multiple = 2, min = 4) @NonNull float[] points, int offset, int length) {                            
                         ^                                                                                                         
  symbol:   class Size                                                                                                             
  location: class CanvasPath                                                                                                       
F:\my-workspace\ohoapp\platforms\tempPlugin\ui_canvas\src\main\java\com\akylas\canvas\CanvasPath.java:49: error: cannot find symbol
   public void setLines(@Size(multiple = 2, min = 4) @NonNull float[] points, int offset, int length) {                            
                                                      ^                                                                            
  symbol:   class NonNull                                                                                                          
  location: class CanvasPath                                                                                                       
F:\my-workspace\ohoapp\platforms\tempPlugin\ui_canvas\src\main\java\com\akylas\canvas\CanvasPath.java:52: error: cannot find symbol
   public void setLines(@Size(multiple = 2, min = 4) @NonNull float[] points, int offset, int length, boolean close) {             
                         ^                                                                                                         
  symbol:   class Size                                                                                                             
  location: class CanvasPath                                                                                                       
F:\my-workspace\ohoapp\platforms\tempPlugin\ui_canvas\src\main\java\com\akylas\canvas\CanvasPath.java:52: error: cannot find symbol
   public void setLines(@Size(multiple = 2, min = 4) @NonNull float[] points, int offset, int length, boolean close) {             
                                                      ^                                                                            
  symbol:   class NonNull                                                                                                          
  location: class CanvasPath                                                                                                       
F:\my-workspace\ohoapp\platforms\tempPlugin\ui_canvas\src\main\java\com\akylas\canvas\CanvasPath.java:57: error: cannot find symbol
   public void addCubicLines(@Size(multiple = 2, min = 8) @NonNull float[] points) {                                               
                              ^                                                                                                    
  symbol:   class Size                                                                                                             
  location: class CanvasPath                                                                                                       
F:\my-workspace\ohoapp\platforms\tempPlugin\ui_canvas\src\main\java\com\akylas\canvas\CanvasPath.java:57: error: cannot find symbol
   public void addCubicLines(@Size(multiple = 2, min = 8) @NonNull float[] points) {                                               
                                                           ^                                                                       
  symbol:   class NonNull                                                                                                          
  location: class CanvasPath                                                                                                       
F:\my-workspace\ohoapp\platforms\tempPlugin\ui_canvas\src\main\java\com\akylas\canvas\CanvasPath.java:61: error: cannot find symbol
   public void addCubicLines(@Size(multiple = 2, min = 8) @NonNull float[] points, int offset) {                                   
                              ^                                                                                                    
  symbol:   class Size                                                                                                             
  location: class CanvasPath                                                                                                       
F:\my-workspace\ohoapp\platforms\tempPlugin\ui_canvas\src\main\java\com\akylas\canvas\CanvasPath.java:61: error: cannot find symbol
   public void addCubicLines(@Size(multiple = 2, min = 8) @NonNull float[] points, int offset) {                                   
                                                           ^                                                                       
  symbol:   class NonNull                                                                                                          
  location: class CanvasPath                                                                                                       
F:\my-workspace\ohoapp\platforms\tempPlugin\ui_canvas\src\main\java\com\akylas\canvas\CanvasPath.java:64: error: cannot find symbol
   public void addCubicLines(@Size(multiple = 2, min = 8) @NonNull float[] points, int offset, int length) {                       
                              ^                                                                                                    
  symbol:   class Size                                                                                                             
  location: class CanvasPath                                                                                                       
F:\my-workspace\ohoapp\platforms\tempPlugin\ui_canvas\src\main\java\com\akylas\canvas\CanvasPath.java:64: error: cannot find symbol
   public void addCubicLines(@Size(multiple = 2, min = 8) @NonNull float[] points, int offset, int length) {                       
                                                           ^                                                                       
  symbol:   class NonNull                                                                                                          
  location: class CanvasPath                                                                                                       
F:\my-workspace\ohoapp\platforms\tempPlugin\ui_canvas\src\main\java\com\akylas\canvas\CanvasPath.java:68: error: cannot find symbol
   public void addCubicLines(@Size(multiple = 2, min = 8) @NonNull float[] points, int offset, int length, boolean close) {        
                              ^                                                                                                    
  symbol:   class Size                                                                                                             
  location: class CanvasPath                                                                                                       
F:\my-workspace\ohoapp\platforms\tempPlugin\ui_canvas\src\main\java\com\akylas\canvas\CanvasPath.java:68: error: cannot find symbol
   public void addCubicLines(@Size(multiple = 2, min = 8) @NonNull float[] points, int offset, int length, boolean close) {        
                                                           ^                                                                       
  symbol:   class NonNull                                                                                                          
  location: class CanvasPath                                                                                                       
F:\my-workspace\ohoapp\platforms\tempPlugin\ui_canvas\src\main\java\com\akylas\canvas\CanvasPath.java:78: error: cannot find symbol
   public void setCubicLines(@Size(multiple = 2, min = 8) @NonNull float[] points) {                                               
                              ^                                                                                                    
  symbol:   class Size                                                                                                             
  location: class CanvasPath                                                                                                       
F:\my-workspace\ohoapp\platforms\tempPlugin\ui_canvas\src\main\java\com\akylas\canvas\CanvasPath.java:78: error: cannot find symbol
   public void setCubicLines(@Size(multiple = 2, min = 8) @NonNull float[] points) {                                               
                                                           ^                                                                       
  symbol:   class NonNull                                                                                                          
  location: class CanvasPath                                                                                                       
F:\my-workspace\ohoapp\platforms\tempPlugin\ui_canvas\src\main\java\com\akylas\canvas\CanvasPath.java:81: error: cannot find symbol
   public void setCubicLines(@Size(multiple = 2, min = 8) @NonNull float[] points, int offset, int length) {                       
                              ^                                                                                                    
  symbol:   class Size                                                                                                             
  location: class CanvasPath                                                                                                       
F:\my-workspace\ohoapp\platforms\tempPlugin\ui_canvas\src\main\java\com\akylas\canvas\CanvasPath.java:81: error: cannot find symbol
   public void setCubicLines(@Size(multiple = 2, min = 8) @NonNull float[] points, int offset, int length) {                       
                                                           ^                                                                       
  symbol:   class NonNull                                                                                                          
  location: class CanvasPath                                                                                                       
F:\my-workspace\ohoapp\platforms\tempPlugin\ui_canvas\src\main\java\com\akylas\canvas\CanvasPath.java:84: error: cannot find symbol
   public void setCubicLines(@Size(multiple = 2, min = 8) @NonNull float[] points, int offset, int length, boolean close) {        
                              ^                                                                                                    
  symbol:   class Size                                                                                                             
  location: class CanvasPath                                                                                                       
F:\my-workspace\ohoapp\platforms\tempPlugin\ui_canvas\src\main\java\com\akylas\canvas\CanvasPath.java:84: error: cannot find symbol
   public void setCubicLines(@Size(multiple = 2, min = 8) @NonNull float[] points, int offset, int length, boolean close) {        
                                                           ^                                                                       
  symbol:   class NonNull                                                                                                          
  location: class CanvasPath                                                                                                       
33 errors                                                                                                                          
```

**Comment by farfromrefug** [comment]
Damn !!! :D dumb me!

**Comment by farfromrefug** [comment]
@DimitrisRK should be fixed in 4.0.6

**Comment by DimitrisRK** [comment]
@farfromrefug 4.0.6 throws a new error:
```
Gradle build...
         + setting applicationId
         + applying user-defined configuration from F:\my-workspace\ohoapp\App_Resources\Android\app.gradle
         + using android X library androidx.legacy:legacy-support-v4:1.0.0
         + adding nativescript runtime package dependency: nativescript-optimized-with-inspector
         + adding aar plugin dependency: F:\my-workspace\ohoapp\node_modules\@nativescript-community\gesturehandler\platforms\android\gesturehandler.aar
         + adding aar plugin dependency: F:\my-workspace\ohoapp\node_modules\@nativescript-community\ui-canvas\platforms\android\canvas-release.aar
         + adding aar plugin dependency: F:\my-workspace\ohoapp\node_modules\@nativescript-community\ui-canvas\platforms\android\ui_canvas.aar
         + adding aar plugin dependency: F:\my-workspace\ohoapp\node_modules\@nativescript\core\platforms\android\widgets-release.aar

Execution failed for task ':app:checkDebugDuplicateClasses'.
Duplicate class com.akylas.canvas.CanvasPath found in modules jetified-canvas-release-runtime.jar (:canvas-release:) and jetified-ui_canvas-runtime.jar (:ui_canvas:)
Duplicate class com.akylas.canvas.CanvasView found in modules jetified-canvas-release-runtime.jar (:canvas-release:) and jetified-ui_canvas-runtime.jar (:ui_canvas:)
Duplicate class com.akylas.canvas.DrawListener found in modules jetified-canvas-release-runtime.jar (:canvas-release:) and jetified-ui_canvas-runtime.jar (:ui_canvas:)
Duplicate class com.akylas.canvas.SizeChangedListener found in modules jetified-canvas-release-runtime.jar (:canvas-release:) and jetified-ui_canvas-runtime.jar (:ui_canvas:)

Go to the documentation to learn how to <a href="d.android.com/r/tools/classpath-sync-errors">Fix dependency resolution errors</a>.
```

**Comment by DimitrisRK** [comment]
> @DimitrisRK really sorry, should have tested it :s released 4.0.7, hope it is fine...

No problem, I do these tests in order to help after all.
Problem seems to be gone for good now.
At night, I'm planning to do more tests on canvas using chart plugin. If any issues occurs, I'll let you know.

**Comment by DimitrisRK** [comment]
I'm closing this since no more related issues seem to exist.
